File: wolf.py
from animal import Animal

class Wolf(Animal):
    def __init__(self, world1, x, y):
        super().__init__()
        self.SetStrength(4)
        self.SetInitiative(4)
        self.SetCords(x, y)
        self.SetMove(True)
        self.world = world1

    def getName(self):
        return "WOLF"

    def Drawing(self):
        self.world.buttons[self.GetCords().y -1][self.GetCords().x -1].configure(bg='#40E0D0')
    
    def New(self, text, reference, x, y):
        self.world.GetOrganisms().add(Wolf(reference, x, y))    

File: Wolf.java
import javax.swing.*;

public class Wolf extends Animal{

    Wolf(World world1, int x, int y) {
        SetStrength(4);
        SetInitiative(4);
        SetCords(x, y);
        SetMove(true);
        world = world1;
    }
    @Override
    public String getName() {
        return "WOLF";
    }

    @Override
    public void Drawing() {
        ImageIcon icon = new ImageIcon("src/Icons/wolf.png");
        world.GetButton(GetCords()).setIcon(icon);
    }
}
